andela/gondor-cfh
# #158487956 Integrate TravisCI

#### What does this PR do?
Integrates TravisCI and adds readme badge.

#### Description of Task to be completed?
- Create travis config file to integrate app with TravisCI
- Add build badge to readme file

#### How should this be manually tested?
- Check for the .travis.yml file in the root of the folder.
- Check the readme file to view the TravisCI badge.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158487956

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
